javascript/javascript3/week3/shoppingCart.js

```diff
@@ -0,0 +1,107 @@
+class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+    }
+    convertToCurrency(currency){
```

Comment:
You could also call a currency API to get the correct up to date conversion rates. That's partly what this exercise was about.